### DISCUSSION TITLE
کد های جیمیلم یادم رفته. <Your question or feedback about the GitHub Code Search and Code View products>

### DISCUSSION THREAD

**Discussion by Moammad1713** [community/community#132338] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

جیمیلم فراموش کردم دیگه به اون شمارم دسترسی ندارم چیکار کنم

**Comment by LiteBrite82** [comment]
Hi @Moammad1713 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
